### PR TITLE
Update MRT Core samples' README.md file.

### PR DESCRIPTION
**Pull request by rohanp-msft**
Update the readme file following the addition of new sample apps.

### CODE REVIEW

**Comment by rohanp-msft** [comment]
@axelandrejs @huichen123 @jefgen 
Ping for this.
Now that the two unpackaged sample apps are now checked in, it's good to update this.

File: MrtCore/README.md [hunk 1]
```diff
@@ -2,22 +2,23 @@
...
+MRT Core is the modern resource loading library used in WinUI 3 and Project Reunion. These samples demonstrate how to use MRT Core to load resources from multiple resource files and how to select which version of a specific named resource to load. 
 
-## Building and running the sample
+## Building and running any of the samples
 
 Before building the sample, make sure to set up your environment correctly by following the steps [here](https://docs.microsoft.com/en-us/windows/apps/project-reunion#set-up-your-development-environment).
+For the unpackaged sample apps, in addition to the steps for setting up the environment for Project Reunion, make sure to follow the instructions [here](https://docs.microsoft.com/en-us/windows/apps/project-reunion/deploy-unpackaged-apps).
 
-1. Open winui_desktop_packaged_app\winui_desktop_packaged_app.sln in Visual Studio 2019 Preview.
+1. Open the solution file (.sln, e.g., winui_desktop_packaged_app\winui_desktop_packaged_app.sln) in Visual Studio 2019 Preview.
```
Comment (jefgen) [inline MrtCore/README.md:18]:
```suggestion
1. Open the solution file (*.sln, e.g., `winui_desktop_packaged_app\winui_desktop_packaged_app.sln`) in Visual Studio 2019 Preview.
```

**Review by jefgen — APPROVED** [review]
fwiw, lgtm.

**Review by jefgen — APPROVED** [review]
:shipit:

File: MrtCore/README.md [hunk 2]
```diff
@@ -2,22 +2,23 @@
 title: "Load resources using MRT Core"
 description: "Use MRT Core's ResourceLoader and ResourceManager to load resources from several resource files"
 page_type: sample
-languages: csharp
+languages: csharp, cpp
```
Comment (huichen123) [inline MrtCore/README.md:5]: >csharp, cpp [](http://example.com/codeflow?start=11&length=11)

why not "C#, C++"?

**Review by huichen123 — APPROVED** [review]
:shipit:

**Pull request merged**